### DISCUSSION TITLE
Sunbird cokreat- contribution service

### DISCUSSION THREAD

**Discussion by agamdahiya** [Sunbird-coKreat/Community#61] [Q&A]
Configuration details - from where will we get the environment variable values mentioned in the below screen shot ?
any other value needs to be updated other than below screen shot? 

![image](https://user-images.githubusercontent.com/127106035/227215806-1a23a2bc-2d8d-451a-8e7d-9b782b3a8b08.png)

**Comment by vaivk369** [comment]
The baseURL should be updated to your coKreat instance host and the SUNBIRD_URL should be pointing to the your sunbird-ed portal host.
For e.g, we have coKreat instance at https://dock.sunbirded.org/ and  sunbird-ed portal at https://dev.sunbirded.org/. so these values in env file looks like

baseURL: process.env.dock_base_url || 'https://dock.sunbirded.org',
SUNBIRD_URL: process.env.sunbird_base_url || 'https://dev.sunbirded.org',

Also, the user and password of the config parameter should be set to the user details you use to login to user postgres database. You would need to create the database named 'sunbird_programs' or you can create your own database and update the name here.
